client/components/deposits-overview/index.tsx

```diff
@@ -49,9 +49,10 @@ const DepositsOverview: React.FC = () => {
 
 	const availableFunds = overview?.available?.amount ?? 0;
 	const pendingFunds = overview?.pending?.amount ?? 0;
+	const totalFunds = availableFunds + pendingFunds;
 
-	// If the available balance is negative, deposits may be paused.
-	const isNegativeBalanceDepositsPaused = availableFunds < 0;
+	// If the total balance is negative, deposits may be paused.
+	const isNegativeBalanceDepositsPaused = totalFunds < 0;
```

Comment:
Simple fix! Though… is it possible that a merchant could have deposits paused because their available balance is negative (large recent dispute/refund) yet with a positive total balance due to large recent purchases in pending? If so we might need to take care.

Reply:
> Though… is it possible that a merchant could have deposits paused because their available balance is negative (large recent dispute/refund) yet with a positive total balance due to large recent purchases in pending?

If this was the case. Pending would move to Available within a few days restoring the available balance and the standard copy would suffice ie `Available funds are automatically dispatched every day`

Reply:
Sounds good to me :)